verifier/src/report_body.rs

```diff
@@ -236,4 +295,24 @@ mod test {
 
         assert_eq!(verifier.verify(&config_id).is_some().unwrap_u8(), 1);
     }
+
+    #[parameterized(
+    equal = { 10, 10 },
+    greater = { 12, 11 },
+    much_greater = { 20, 1 },
```

Comment:
```suggestion
        equal = { 10, 10 },
        greater = { 12, 11 },
        much_greater = { 20, 1 },
```

Reply:
I realize this is kind of a pointless pet peeve / disappointment in `rustfmt`, buuut...